src/sentry/static/sentry/app/stores/organizationStore.jsx

```diff
@@ -9,20 +11,36 @@ const OrganizationStore = Reflux.createStore({
     this.listenTo(OrganizationActions.update, this.onUpdate);
     this.listenTo(OrganizationActions.fetchOrg, this.reset);
     this.listenTo(OrganizationActions.fetchOrgError, this.onFetchOrgError);
+
+    // mark the store as dirty if projects or teams change
+    this.listenTo(ProjectActions.createSuccess, this.onProjectOrTeamChange);
+    this.listenTo(ProjectActions.updateSuccess, this.onProjectOrTeamChange);
+    this.listenTo(ProjectActions.loadStatsSuccess, this.onProjectOrTeamChange);
```

Comment:
Do we need to mark as dirty for this action?

Reply:
Oh yea this probably doesn't need to be there